### PR TITLE
Base 64 encode before decoding [WIP]

### PR DESCRIPTION
**Pull request by qasimzee**
Issue: https://github.com/celery/kombu/issues/1212

### CODE REVIEW

File: kombu/transport/SQS.py [hunk 1]
```diff
@@ -284,6 +284,8 @@ def _message_to_python(self, message, queue_name, queue):
             body = base64.b64decode(message['Body'].encode())
         except TypeError:
             body = message['Body'].encode()
+        except base64.binascii.Error:
+            body = base64.b64decode(base64.b64encode(message['Body'].encode()))
```
Comment (matusvalo) [inline kombu/transport/SQS.py:288]: Do we need to encode and decode message?

**Comment by auvipy** [comment]
should we close this as per the issue?

**Comment by thedrow** [comment]
I don't understand why your fix solves the problem.

**Comment by matusvalo** [comment]
This PR seems to be inactive and the purpose seems to be unclear. I would close this issue.

**Pull request closed without merging**